src/utils/tests/testDataRequest.ts

```diff
@@ -15,3 +15,27 @@ export const testDataRequest = {
   eventIds: ['123', '456'],
   piiTypes: ['passport_number']
 } as DataRequestParams
+
+export const testDataRequestWithValues = {
```

Comment:
would testDataRequestWithAllValues be more descriptive (to differentiate it from the var above)?

Reply:
Yep, have done that now